ansible_collections/arista/avd/roles/eos_designs/doc/fabric-topology.md

```diff
@@ -344,3 +344,31 @@ defaults <- node_group <- node_group.node <- node
     # VLAN number assigned to Inband Management SVI on l2leafs in default VRF.
     inband_management_vlan: < vlan-id | Default -> 4092 >
 ```
+
+### EVPN Multihoming
+
+```yaml
```

Comment:
These new options are not specific to EVPN Multihoming, so it should be moved or renamed to a more generic heading

Reply:
I can move them to two groups. Link-tracking and Lacp port-id and mention there that these knobs are useful for multihoming.

Reply:
Used the heading ```Other Optional Settings```.

Reply:
I suggest to move it up under the genereric section.